src/App.tsx

```diff
@@ -1,14 +1,15 @@
 import { SearchBar } from "./components/SearchBar"
 import { CarAttributes } from "./components/CarAttributes.tsx"
+import { MainLogo } from "./components/MainLogo.tsx"
+import { ColorsLegend } from "./components/ColorsLegend.tsx"
```

Comment:
(✂️ file extensions)
```suggestion
import { MainLogo } from "./components/MainLogo"
import { ColorsLegend } from "./components/ColorsLegend"
```